From f6b44f84094ed463b265aafaa4b16611b1d5ba13 Mon Sep 17 00:00:00 2001
From: Alex Wilson <alex.wilson@joyent.com>
Date: Mon, 5 Jun 2017 11:50:42 -0700
Subject: [PATCH] joyent/node-smartdc-auth#12 KeyRingHomeDirPlugin.listKeys()
 invokes callback twice on failure joyent/node-smartdc-auth#10 bump the
 signaturecache event emitter limit

---
 CHANGES.md   | 6 +++++-
 package.json | 4 ++--
 2 files changed, 7 insertions(+), 3 deletions(-)

diff --git a/CHANGES.md b/CHANGES.md
index 13266c4..50c0730 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -4,11 +4,15 @@ Known issues:
 
 - `triton ssh ...` disables ssh ControlMaster to avoid issue #52.
 
-
 ## not yet released
 
+## 5.2.1
+
 - [joyent/node-triton#193] Fix possible CLI crash with `triton ssh ...` if the
   instance's image doesn't have any tags.
+- [joyent/node-smartdc-auth#10] Avoid spurious warnings on console
+- [joyent/node-smartdc-auth#12] Fix possible crash in authentication due to 
+  twice-called callback
 
 ## 5.2.0
 
diff --git a/package.json b/package.json
index bff9b76..be5945f 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "triton",
   "description": "Joyent Triton CLI and client (https://www.joyent.com/triton)",
-  "version": "5.2.0",
+  "version": "5.2.1",
   "author": "Joyent (joyent.com)",
   "homepage": "https://github.com/joyent/node-triton",
   "dependencies": {
@@ -20,7 +20,7 @@
     "restify-errors": "3.0.0",
     "rimraf": "2.4.4",
     "semver": "5.1.0",
-    "smartdc-auth": "2.5.2",
+    "smartdc-auth": "2.5.5",
     "sshpk": "1.10.2",
     "sshpk-agent": "1.4.2",
     "strsplit": "1.0.0",
-- 
2.21.0

